psi4/src/psi4/lib3index/dfhelper.h

```diff
@@ -151,7 +159,21 @@ class PSI_API DFHelper {
     /// @param omega double indicating parameter for other type
     ///
     void set_omega(double omega) { omega_ = omega; }
-    size_t get_omega() { return omega_; }
+    double get_omega() { return omega_; }
```

Comment:
omegas can be negative?

Reply:
The `size_t` was a typo. The double valued function is the one currently in the code.